src/test/shell/bazel/BUILD

```diff
@@ -767,6 +767,14 @@ sh_test(
     tags = ["no_windows"],
 )
 
+sh_test(
+    name = "tags_propagation_native_test",
```

Comment:
@hlopko should this test be part of the bazel_rules_cc_test.sh?

Reply:
I actually think we don't need an integration test for this, wdyt about adding an unit test to CcLibraryConfiguredTargetTest? We're testing analysis-only behavior anyway. You'll assert that the action in constructed as expected.

Reply:
(and also a test to CcBinaryConfiguredTargetTest, GenruleConfiguredTargetTest...)

Reply:
hm, we could do that. I just prefer integration tests in this case - to make sure that in 'real' env it actually works. I also find them more readable here :)

Reply:
Well, you find them more readable because you didn't spend much time in the area :) We actually like our BuildViewTestCases more becasue they allow us to inspect the state of the world more easily. Instead of comparing handwritten line of text that aquery outputs you can have assertThat(executionInfo).containsExactly(...).

Plus the integration test will have exactly same coverage as the BuildViewTestCase.